src/anm/animation/element.js

```diff
@@ -829,15 +847,8 @@ Element.prototype.render = function(ctx, gtime, dt) {
 
             var mask = this.$mask;
 
-            // FIXME: move this chain completely into one method, or,
-            //        which is even better, make all these checks to be modifiers
-            // FIXME: call modifiers once for one moment of time. If there are several
-            //        masked elements, they will be called that number of times
-            if (!(mask.fits(ltime) &&
-                  mask.matchesSwitch() &&
-                  mask.modifiers(ltime, dt) &&
-                  mask.visible)) return;
-                  // what should happen if mask doesn't fit in time?
+            var mask_ltime = mask.ltime(gtime),
```

Comment:
Here the variables from [line 825](https://github.com/Animatron/player/pull/312/files#diff-a072a28291a97d7ac72e402357d42fc6R825) do reappear, including `var mask` in lines [848](https://github.com/Animatron/player/pull/312/files#diff-a072a28291a97d7ac72e402357d42fc6R848) and [821](https://github.com/Animatron/player/pull/312/files#diff-a072a28291a97d7ac72e402357d42fc6R821). I've fixed that in 6e71df7573